.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+# <!-- Use the title to describe PR changes in the imperative mood --> #
+
+## 🗣 Description ##
+
+<!-- Describe the "what" of your changes in detail. -->
+<!-- To avoid scope creep, limit changes to a single goal. -->
+
+## 💭 Motivation and context ##
+
+<!-- Why is this change required? -->
+<!-- What problem does this change solve? How did you solve it? -->
+<!-- What isn't included in this PR? -->
+<!-- Mention any related issue(s) here to cross-link automatically. -->
+
+## 🧪 Testing ##
+
```

Comment:
<!-- Please provide a link to the Netlify Preview URL (if necessary). -->

Reasoning: The project that I am on is intended to be viewed on mobile. To view the preview, you must navigate to the  bottom of the pull page in the browser (@netlify deploy/netlify — Deploy Preview ready!).  When viewing on a mobile device, the person reviewing is not able to access that link as it is not included in the GitHub mobile app. In the testing comment section, most of my PRs require me to copy the preview URL and paste in said location. I am then able to switch to my mobile GitHub app and view the preview.  

I believe that it would be appropriate to require (if app is to be viewed on a mobile device) the link posted in the testing section so the reviewer may view on the device intended. This might be covered on lines 19 and 20, but in a growing organization, I believe being specific is helpful.

Reply:
Definitely - thanks for the suggestion and your reasoning.

Reply:
Updated in https://github.com/trabian/.github/pull/1/commits/301a333e6e212f3573189359bf03e53571d68a47